src/vma/sock/sockinfo.h

```diff
@@ -61,6 +61,8 @@
 #define BYTE_TO_KB(byte_value)		((byte_value) / 125)
```

Comment:
migth be not needed `SI_RX_EPFD_EVENT_MAX`

Reply:
Yes, thanks. It doesn't need